manifests/unit_file.pp

```diff
@@ -4,7 +4,7 @@
 #
 # @see systemd.unit(5)
 #
-# @param name [Pattern['^[^/]+\.(service|socket|device|mount|automount|swap|target|path|timer|slice|scope)$']]
+# @param name [Pattern['^[^/]+\.(service|socket|device|mount|automount|swap|target|path|timer|slice|scope|kube)$']]
```

Comment:
I think we shouldn't document it here, that's already in the type alias definition.